lib/AlternativeNamesFieldArray/AlternativeNamesFieldArray.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+
+import {
+  Button,
+  Col,
+  Headline,
+  IconButton,
+  TextField,
+  Row,
+} from '@folio/stripes/components';
+
+import withKiwtFieldArray from '../withKiwtFieldArray';
+
+class AlternativeNamesFieldArray extends React.Component {
+    static propTypes = {
+      items: PropTypes.arrayOf(PropTypes.object),
+      name: PropTypes.string.isRequired,
+      onAddField: PropTypes.func.isRequired,
+      onDeleteField: PropTypes.func.isRequired,
+    }
+
+    static defaultProps = {
+      items: [],
+    }
+
+    renderNames = () => {
+      const { items, name, onDeleteField } = this.props;
+      return items.map((item, index) => (
+        <div
+          key={index}
+          data-test-render-alternative-names
+        >
+          <Row>
+            <Col xs={11}>
+              <Field
+                autoFocus
```

Comment:
Are you sure about this? What happens when you open up an agreement for editing that already has some alternative names? We _shouldn't_ be autofocusing this field.